test.py

```diff
@@ -0,0 +1,41 @@
+from docetl.operations.code_operations import CodeMapOperation
```

Comment:
the tests themselves seem fine but can we put them in the tests folder & use pytest? maybe a test_code_ops.py?

Reply:
done